docs-temp-for-PR/Documentation-Dos-And-Do-Nots.md

```diff
@@ -0,0 +1,40 @@
+When writing documentation - be it source-code documentation, wiki content, etc. - the following are best practices to keep in mind.
+
+## Write from the second person
+
+A consistent point of view helps documentation flow nicely.
+And since our docs are aimed at some potential developer or customer, it is nice to address them directly.
+
+## Only write about supported functionality
+
+Plans change.
+Even if we intend to add support for some scenario in the future, it may not come to fruition.
+It is best to keep our documentation scoped to only the things we support *right now*.
+
+Note that it's okay to discuss *ideas* for future features or changes, but it needs to be made clear that this is not a promise.
+
+## Don't introduce new terminology
+
+It is imperative that we keep the Fluid Framework's barrier to entry as low as possible to make adoption easier.
+Please avoid adding new terminology whenever possible.
+If you think you need to add something new to describe some new concept, it will need to be reviewed by the team.
```

Comment:
Would be great to provide as much guidance here as possible. Maybe even an alias for now?

Reply:
@skylerjokiel That sounds like a great idea. Any suggestions for such an alias? Or should we just point folks to the github discussions space? Or both?